python/cuml/test/test_random_forest.py

```diff
@@ -317,12 +319,12 @@ def test_rf_classification_float64(datatype, column_info, nrows):
 @pytest.mark.parametrize('column_info', [unit_param([20, 10]),
                          quality_param([200, 100]),
                          stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(500), quality_param(5000),
+@pytest.mark.parametrize('nrows', [unit_param(5000), quality_param(5000),
```

Comment:
maybe we should have different values for unit_param and quality_param

Reply:
What is the purpose of having different unit and quality in this test in particular?

Reply:
(Open question, we might need a more clear guideline regarding that)

Reply:
Can we have just a unit test amount? I understand the goal of stress tests but don't feel very clear on unit vs. quality. I'd hope that unit tests are a good way to check quality! ;)

Reply:
We generally run the quality tests with a larger dataset in all the other tests as this helps us to test the performance of cuml vs sklearn for larger datasets. In addition, when we run all the tests (unit, quality and stress) we will be running the same tests twice.